unkeyed/unkey
# fix(web): change `ApisOverviewPage` upsell wording

## What does this PR do?



Fixes #1393 

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

<img width="1424" alt="Screenshot 2024-04-15 at 10 41 55 PM" src="https://github.com/unkeyed/unkey/assets/51177379/dd247006-b361-4c98-a26a-850d7b600231">




## Type of change




## How should this be tested?



- Test A: Create a workspace in an organization that doesn't have a paid plan. Go to the APIs section and you should see the new copy.

## Checklist



### Required


### Appreciated
